src/OpenTelemetry.Extensions/OpenTelemetry.Extensions.csproj

```diff
@@ -16,7 +16,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenTelemetry" Version="1.1.0" />
+    <PackageReference Include="OpenTelemetry" Version="1.4.0-beta.3.8" />
```

Comment:
There is not OpenTelemetry 1.4.0-beta.3.8 on the nuget side.

The latest stable is 1.3.1, the latest beta is 1.4.0-beta.1.

See: https://www.nuget.org/packages/OpenTelemetry/